src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs

```diff
@@ -1188,7 +1188,7 @@ void Goo()
 }");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.DebuggingBreakpoints)]
```

Comment:
the trait is on hte class (and it thus inherited by all members inside).  removed these two stragglers.